.github/workflows/build.yml

```diff
@@ -162,7 +162,7 @@ jobs:
 
   check-documentation:
     name: Check documentation
-    runs-on: ubuntu-latest
+    runs-on: macOS-latest
```

Comment:
can you replace please `uses: actions/upload-artifact@v2` -> `uses: actions/upload-artifact@v3` in 2 places

Reply:
Done